src/pages/SignUp.vue

```diff
@@ -98,12 +99,17 @@
 </template>
 
 <script>
-import { mapActions } from "pinia";
+import { mapActions, mapState } from "pinia";
 
 import { useUserStore } from "@/store/userStore.js";
 import { useAuthStore } from "@/store/authStore.js";
 
+import GLoginButton from "@/components/GLoginButton.vue";
+
 export default {
+  components: {
+    "g-login": GLoginButton
```

Comment:
need to figure out how we going to save the session token or google auth on the backend